xenserver/resource_vm.go

```diff
@@ -346,6 +378,9 @@ func resourceVMCreate(d *schema.ResourceData, m interface{}) error {
 		}
 	}
 
+	d.Partial(false)
+
+	// TODO: Seems like this is more about the state of the resource than the creation of the resource?
```

Comment:
You are correct. However it is more practical to startup VM so other tasks could be performed on it, for example provisioning. But maybe it would be practical to introduce new field to the resource that will indicate desired VM state